web/src/components/njwds-layout/SidebarPageLayout.tsx

```diff
@@ -22,6 +23,7 @@ export const SidebarPageLayout = ({
   stackNav,
   belowBoxComponent,
   titleOverColumns,
+  nonWrappingLeftColumn,
 }: SidebarPageLayoutProps): ReactElement => {
   const isLargeScreen = useMediaQuery(MediaQueries.desktopAndUp);
```

Comment:
[dust] How much of a lift would it be to remove this `useMediaQuery()` as part of this ticket?